xeda/xeda_app.py

```diff
@@ -185,11 +208,13 @@ def __call__(self, parser, args, value, option_string=None):
                 #     sys.exit(f'Flow {flow_name} not found')
                 setattr(args, self.dest, flow_name)
 
-        parser.add_argument('flow', metavar='[RUNNER_NAME:]FLOW_NAME', action=CommandAction,
-                            help=f'Flow name optionally prepended by flow-runner.' +
-                            'If runner is not specified the default runner is used.\n' +
-                            f'Available flows are: {registered_flows}\n' +
-                            f'Available runners are: {[camelcase_to_snakecase(n) for n, _ in self.runner_classes]}'
+        parser.add_argument('flow', choices=['init','[RUNNER_NAME:]FLOW_NAME'], action=CommandAction,
```

Comment:
Maybe a better approach would be to implement init as a flow and handle it as a special case in the runner. Maybe add a class of "special flows", i.e., flows that do not run in flow_run_dir, do not generate results, do not dump settings etc?

Reply:
I thought about doing that, but decided against it because it didn't really match the word "flow" semantically. If you think it would be better then I can do it that way